unitary/quantum_chess/circuit_transformer.py

```diff
@@ -76,7 +76,9 @@ def qubits_within(
             c += self.qubits_within(depth - 1, qubit + diff, qubit_list, visited)
         return c
 
-    def find_start_qubit(self, qubit_list: List[cirq.Qid], depth=3) -> cirq.GridQubit:
+    def find_start_qubit(
+        self, qubit_list: Iterable[cirq.Qid], depth=3
```

Comment:
`Iterable` is too loose since you need to be able to iterate it more than once. `Sequence` I guess